thorlcr/activities/thdiskbaseslave.cpp

```diff
@@ -318,15 +318,18 @@ IThorRowInterfaces * CDiskReadSlaveActivityBase::queryProjectedDiskRowInterfaces
 void CDiskReadSlaveActivityBase::serializeStats(MemoryBuffer &mb)
 {
     CRuntimeStatisticCollection activePartStats(diskReadPartStatistics);
```

Comment:
minor: would be good to move this into the if block, as unused otherwise and avoids the expense of this ctor.

Reply:
Any problems putting it inside if (partHandler)? or should it go in the outer if?

Reply:
would be fine to go in the inner, but personally I'd leave inside the outer,
because I think the if (partHandler) is probably redundant, afaics, it must be non-null, by the time this is called (but I'd leave it as is and not conflate that with this fix)